### DISCUSSION TITLE
bullmq use problem

### DISCUSSION THREAD

**Discussion by chenhuanguang** [taskforcesh/bullmq#1886] [Q&A]
1. I used kue in my project before, but now I want to use bullmq. The redis library I used before is "redis". Do I need to replace it with "ioredis" library? Is bullmq compatible?
2. Which one is recommended for bull and bullmq? The requirement of my project is to require higher stability

**Comment by roggervalf** [comment]
Hi @chenhuanguang, you should use ioredis, redis package is not compatible
